checklists/alz_checklist.en.json

```diff
@@ -2480,22 +2480,78 @@
         {
             "category": "Security",
             "subcategory": "Operations",
-            "text": "Connect default resource configurations to a centralized Azure Monitor Log Analytics workspace.",
+            "text": "Creating dedicated workspaces for Monitor and Secuity to to segregate ownership of data between operational and security teams and optimize costs.",
             "waf": "Security",
             "service": "Monitor",
             "guid": "e5f8d79f-2e87-4768-924c-516775c6ea95",
             "id": "G03.08",
-            "severity": "Medium",
+            "severity": "High",
             "link": "https://learn.microsoft.com/azure/azure-monitor/logs/design-logs-deployment"
         },
+        {
+            "category": "Security",
+            "subcategory": "Operations",
+            "text": "Implement SIEM solution to consolidate security data in a central location where it can be correlated across various services.",
```

Comment:
We need to make this more specific.  If we are advocating for Sentinel, can we specify that the customer should deploy Sentinel as their SIEM and use the platform Log Analytics Workspace for that?

Reply:
ok, I will change it to: "Implement Sentinel as SIEM solution to consolidate security data in a central location where it can be correlated across various services. Deploy Log analytic workspace as Sentinel workspace enabled for Sentinel."

Reply:
Also, can we get a link to the CAF Ready documentation that suggests this explicitly?  If it is not part of the CAF Ready Landing Zone, then it might be better to keep this just in the Security Checklist instead.

Reply:
Its appear here: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/get-started/security#step-3-develop-a-security-plan, under: Security operations center (SOC) reference architecture to modernize security detection, response, and recovery.
Is it better?

Reply:
That section is from the CAF Secure page - in line with the other conversation, I am thinking it probably makes sense to make a checklist for CAF Secure to "partner" with this CAF Ready guidance.  Then CAF Ready/ALZ can have an item to "Use the CAF Secure checklist to..." as a pointer.